src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ConditionalOrder.java

```diff
@@ -212,6 +212,17 @@ public class ConditionalOrder {
     )
     private List<ListValue<ClarificationResponse>> clarificationResponsesSubmitted;
 
+    @CCD(
+        label = "Service Confirmed"
+    )
+    private YesOrNo serviceConfirmed;
+
+    @CCD(
+        label = "Documents uploaded for Proof of Service ",
```

Comment:
Do we need a space after Service?

Reply:
Removed in commit 
https://github.com/hmcts/nfdiv-case-api/pull/2157/commits/e7b0e3f9160f42aa3d615468c0e91ee890aded23